dash/backend/src/modules/falco/controllers/falco.controller.ts

```diff
@@ -88,27 +106,48 @@ export class FalcoController {
 
 
     @Post(':clusterid/create')
-    // @TODO: Update to use same authorization method as kube-bench and kube-hunter
     async createFalcoLog(
         @Param('clusterid') clusterId: number,
         @Body() falcoLog: FalcoWebhookInputDto,
         @Query('key') key: string
-    ): Promise<number> {
-
-        //find user authority by user's apikey
+    ): Promise<FalcoDto> {
+        // look up user by api key
         const currentUserAuthObj = await this.userDao.loadUserByApiKey(key);
-        // if the current user's api is valid
-        if (currentUserAuthObj !== null && currentUserAuthObj != undefined ) {
-            // get all authorities from the current user
-            const currentUserAuth = currentUserAuthObj[0].authorities;
-            // get Falco authority from Authorityid
-            let authorityArr: AuthorityId [] = [AuthorityId.FALCO];
-            // is the user a Falco user
-            let isFalcoUser = this.authService.checkAuthority(currentUserAuth, authorityArr);
-            if (isFalcoUser) {
-                return this.falcoService.createFalcoLog(clusterId, falcoLog);
-            }
+        if (!currentUserAuthObj) {
+            throw new UnauthorizedException('Access Denied!');
+            return;
         }
+
+        // get all authorities from the current user
+        const currentUserAuth = currentUserAuthObj[0].authorities;
+        let isFalcoUser = this.authService.checkAuthority(currentUserAuth, [AuthorityId.FALCO]);
+        if (!isFalcoUser) {
+            throw new UnauthorizedException('Access Denied!');
+            return;
+        }
+
+        // check if falco settings rules should be applied
+        /*const rules = getFalcoSettingsRules(clusterId);
```

Comment:
Needs a comment explaining why this code is here

        // @TODO: Will implement falco filters/rules settings at a later time